tests/Fixtures/TestBundle/Entity/Dummy.php

```diff
@@ -130,6 +140,13 @@ class Dummy
      */
     public $nameConverted;
 
+    /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean", nullable=true, name="existsd")
```

Comment:
@soyuka Do you think we should get another PR merged into 2.0 branch to make some of the other filters more strict in terms of which properties they can be applied to? I'd argue that it should not be considered a BC break, for example if the RangeFilter stopped working for boolean properties (lol) or if the SearchFilter stopped working with a numeric property (really really bad idea in the first place).